ConnectorGrasshopper/ConnectorGrasshopperShared/Streams/Deprecated/StreamListComponent.cs

```diff
@@ -109,12 +115,12 @@ public override void SolveInstanceWithLogContext(IGH_DataAccess DA)
         {
           var client = new Client(account);
           // Save the result
-          var result = await client.StreamsGet(limit);
+          var result = await client.StreamsGet(limit).ConfigureAwait(false);
           streams = result
             .Select(stream => new StreamWrapper(stream.id, account.userInfo.id, account.serverInfo.url))
             .ToList();
         }
-        catch (Exception e)
+        catch (SpeckleGraphQLException e)
```

Comment:
One of the tickets r.e. Core is ensuring that the client functions have clear exception paths. I'm 70% they will only throw `SpeckleGraphQlExceptions`. But I'm not 100% sure, since we've definitely   seen some suspicious null responses from the server.

Reply:
I'm reverting this since this class is obsolete. No need to change it.